platform.md

```diff
@@ -133,6 +122,410 @@ A platform MAY support any number of mixins for a given stack in order to suppor
+| `<buildpacks>`  | `CNB_BUILDPACKS_DIR`  | `/cnb/buildpacks` | Path to buildpacks directory (see [Buildpacks Directory Layout](#buildpacks-directory-layout))
+| `<group>`       | `CNB_GROUP_PATH`      | `./group.toml`    | Path to output group file
+| `<log-level>`   | `CNB_LOG_LEVEL`       | `info`            | Log Level
+| `<order>`       | `CNB_ORDER_PATH`      | `./order.toml`    | Path to order definition (see [`order.toml` (TOML)](#order.toml-(toml)))
+| `<plan>`        | `CNB_PLAN_PATH`       | `./plan.toml`     | Path to output build plan file
+| `<platform>`    | `CNB_PLATFORM_DIR`    | `/platform`       | Path to platform directory
+
+| Output             | Description
+|--------------------|----------------------------------------------
+| [exit status]      | Success (0), or error (1+)
+| `/dev/stdout`      | Logs (info)
+| `/dev/stderr`      | Logs (warnings, errors)
+| `<group>`          | Detected buildpack group  (see [`group.toml` (TOML)](#group.toml-(toml)))
+| `<plan>`           | Resolved Build Plan (see  data format in [Buildpack Interface Specification](buildpack.md))
+
+#### `analyzer`
+Usage: 
+```
+/cnb/lifecycle/analyzer <image> \
+  [-analyzed <analyzed>] \
+  [-cache-dir <cache-dir>] \
+  [-cache-image <cache-image>] \
+  [-daemon] \ # sets <daemon>
+  [-gid <gid>] \
+  [-group <group>] \
+  [-layers <layers>] \
+  [-skip-layers <skip-layers>] \
+  [-uid <uid>] \
+```
+
+| Input          | Environment Variable  | Default Value     | Description
+|----------------|-----------------------|-------------------|----------------------
+| `<analyzed>`   | `CNB_ANALYZED_PATH`   | `./analyzed.toml` | Path to output analysis metadata (see [`analyzed.toml` (TOML)](#analyzed.toml-(toml))
+| `<cache-dir>`  | `CNB_CACHE_DIR`       |                   | Location of cache, provided as a directory
+| `<cache-image>`| `CNB_CACHE_IMAGE`     |                   | Location of cache, provided as an image
+| `<gid>`        | `CNB_GROUP_ID`        |                   | Group of user that build phase will run as
+| `<group>`      | `CNB_GROUP_PATH`      | `./group.toml`    | Path to group definition (see [`group.toml` (TOML)](#group.toml-(toml)))
+| `<image>`      |                       |                   | Image reference to be analyzed (usually the result of the previous build)
+| `<layers>`     | `CNB_LAYERS_DIR`      | `/layers`         | Path to layer directory
+| `<skip-layers>`| `CNB_SKIP_LAYERS`     | `false`           | Do not write layer metadata
+| `<uid>`        | `CNB_USER_ID`         |                   | User that build phase will run as
+| `<use-daemon>` | `CNB_USE_DAEMON`      | `false`           | Analyze image from docker daemon
+
+IF `<use-daemon>` is `false`, `<image>` MUST be a valid registry reference
+IF `<use-daemon>` is `true`, `<image>` MUST be a valid registry reference OR a docker image ID
+The lifecycle MUST accept valid references to non-existent images without error.
```

Comment:
This reads a bit funnily in markdown (lines all squashed together). Might need to add bullets or something.